src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx

```diff
@@ -439,8 +420,8 @@ class DebugMeta extends React.PureComponent<Props, State> {
 
 export default DebugMeta;
 
-const StyledList = styled(List)<{overflowHidden: boolean; height: number}>`
-  ${p => p.overflowHidden && 'overflow: hidden !important;'}
+const StyledList = styled(List)<{height: number}>`
+  overflow: scroll;
```

Comment:
you don't need to set it here. I believe the list component already sets it to `auto`